source/query-with-natural-language/ai-and-data-usage-information.txt

```diff
@@ -0,0 +1,44 @@
+.. _compass-ai-data-usage:
+
+=============================
+AI and Data Usage Information
+=============================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+
+Querying with natural language in Compass is powered by Generative AI
+(Gen AI), and may give inaccurate responses. Please see our `Generative AI FAQ 
+<https://dochub.mongodb.org/core/faq-ai-features>`__ 
+for more information about Gen AI in MongoDB products.
+
+Third Party Providers
+---------------------
+
+This feature currently uses the `Azure OpenAI Service
+<https://azure.microsoft.com/en-us/products/ai-services/openai-service>`__ 
+hosted by Microsoft. 
+
+How Your Data is Used
+---------------------
+
+When you query with natural language in Compass, the schema of the 
+collection you are querying including the collection name, field names, 
+and types. 
+
+The information that is sent is not shared with any other third 
```

Comment:
(rephrase for clarity)
I stumbled on this a little bit. Maybe slight tweak. Something like:
"The information sent to our AI provider is not shared ..."

Reply:
Updated.